### PR TITLE
Add method to LocaleChanger to check whether or not it has been

### PR DESCRIPTION
**Pull request by hndmrsh**
initialized

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,6 +39,7 @@ allprojects {
         maven {
```
Comment (jamiesanson) [inline build.gradle]: You can get rid of this maven declaration, the `google()` one covers it

File: library/build.gradle [hunk 2]
```diff
@@ -20,12 +20,11 @@ apply plugin: 'com.github.dcendents.android-maven'
...
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 27
 
     defaultConfig {
         minSdkVersion 15
-        targetSdkVersion 25
+        targetSdkVersion 27
```
Comment (jamiesanson) [inline library/build.gradle:27]: Can you target 28 without using support v28? Might be the way to go

**Review by jamiesanson — COMMENTED** [review]
Just a couple of comments, nothing major

File: library/build.gradle (hunk 2 above)
Comment (hndmrsh) [reply library/build.gradle:27]: Nah, Lint gives an error if you try to do this (support library version should match the target SDK)

**Pull request merged**